apps/image-editor/src/css/submenu.styl

```diff
@@ -62,11 +62,12 @@
         label > span
             width: 55px;
             white-space: nowrap;
-    .{prefix}-submenu-align:first-child
+    .{prefix}-submenu-align:first-child, .{prefix}-submenu-align-mr-0
```

Comment:
Where is the style used?
I can't find it.

Reply:
Ah, sorry. It don't used anywhere :)

Reply:
fixed.